package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "admin-ui-bundle",
```

Comment:
Can you call it `netgen-admin-ui-bundle`, for consistency with our other packages?